### PR TITLE
Add `mc-sgx-dcap_quoteverify` crate

### PR DESCRIPTION
**Pull request by nick-mobilecoin**

### CODE REVIEW

**Comment by nick-mobilecoin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #128** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/128" target="_blank">Graphite</a> 
    * **PR #137** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/137" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/137?utm_source=stack-comment).

File: dcap/quoteverify/src/lib.rs
Comment (nick-mobilecoin) [inline dcap/quoteverify/src/lib.rs:6]: Started this in the `lib.rs` module for now, I'm up for suggestions on name for a containing module

File: dcap/quoteverify/src/lib.rs
Comment (awygle) [inline dcap/quoteverify/src/lib.rs:25]: Hm. Is there any Rust idiomatic guidance on global configuration methods like this? It feels a bit sketchy as it's modifying global state, but doing it secretly. I am wondering if we should make some kind of Qv struct to attach this method and the follow-on `load_policy` and `supplemental_data_size` to. That would allow us to mark it as `!Send` (actually I think that happens automatically) and force people to think before pulling the rug out from under another thread.

**Review by awygle — APPROVED** [review]
I think I'm OK with this as it seems the common Rust thing to do is not to worry about the sort of issues I mentioned but we should at least discuss the options before committing too deeply.

**Review by jcape — APPROVED** [review]
I think we really want to fix this, because it _really_ feels like we're leaving a footgun for future users... That said, it doesn't need to get fixed in this PR.

File: dcap/quoteverify/src/lib.rs
Comment (nick-mobilecoin) [reply dcap/quoteverify/src/lib.rs:25]: created #145

**Comment by nick-mobilecoin** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/137?utm_source=stack-comment).


Job Reference: 6Shi6vPuuE79lf4JMQAu

**Pull request merged**
